### ISSUE TITLE
some questions about warmup-updates setting

### ISSUE THREAD

**Issue by multibla** [NVIDIA/DeepLearningExamples#273]
I have some questions about warm up when i look at README in 
https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/Translation/Transformer

In README.md, the training script is as follows:
`python -m torch.distributed.launch --nproc_per_node 8 /workspace/translation/train.py /data/wmt14_en_de_joined_dict \
  --arch transformer_wmt_en_de_big_t2t \
  --share-all-embeddings \
  --optimizer adam \
  --adam-betas '(0.9, 0.997)' \
  --adam-eps "1e-9" \
  --clip-norm 0.0 \
  --lr-scheduler inverse_sqrt \
  --warmup-init-lr 0.0 \
  --warmup-updates 4000 \
  --lr 0.0006 \
  --min-lr 0.0 \
  --dropout 0.1 \
  --weight-decay 0.0 \
  --criterion label_smoothed_cross_entropy \
  --label-smoothing 0.1 \
  --max-tokens 5120 \
  --seed 1 \
  --target-bleu 28.3 \
  --ignore-case \
  --fp16 \
  --save-dir /workspace/checkpoints \
  --distributed-init-method env:// ` 

And the main parameters are:
**max_tokens: 5120    warmup-updates:4000   gpu: 8   fp16**
one batch tokens:  5120 * 8

In the following advices:
![image](https://user-images.githubusercontent.com/35195790/68010362-5ebe6580-fcbf-11e9-8e70-08aa626eac90.png)
the main parameters are:
**max_tokens: 2560  warmup-updates: 8000    gpu:4  update-freq:2  fp32** 
one batch tokens: 2560 * 2 * 4

Now i have some questions about this, the same tokens number is set in one batch,  actually the num-updates are same in one epoch. Why do i need to **double the warmup-updates**?

**Comment by jbaczek** [comment]
Right, that is a mistake in the readme. Thanks for pointing this out!
